ci-operator/step-registry/multiarch/tuning-operator/e2e/olm-kustomize/multiarch-tuning-operator-e2e-olm-kustomize-commands.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+export CLEANUP=true
+# execute the slow tests only on one of the runs (choose randomly)
+if [ $((RANDOM % 100)) -lt 50 ]; then
+  export GINKGO_EXTRA_ARGS="--skip=\\[slow-30\\]"
+  echo "[SKIP] Skipping slow tests in the OLM deployment"
+fi
+USE_OLM=true ./hack/deploy-and-e2e.sh
+if [ "${GINKGO_EXTRA_ARGS}" == "" ]; then
+  export GINKGO_EXTRA_ARGS="--skip=\\[slow-30\\]"
+  echo "[SKIP] Skipping slow tests in the Kustomize deployment"
+fi
```

Comment:
whether we should also reset `GINKGO_EXTRA_ARGS=""` if it has been set in olm case?